deepchecks/base/dataset.py

```diff
@@ -420,6 +441,20 @@ def train_test_split(self,
                                              stratify=stratify)
         return self.copy(train_df), self.copy(test_df)
 
+    @staticmethod
+    def _infer_label_type(
+            label_col: pd.Series,
+            max_categorical_ratio: float,
+            max_categories: int,
+            max_float_categories: int
+    ):
+        if not is_numeric_dtype(label_col):
+            return 'classification label'
```

Comment:
I would remove the "label" word from the type.

Reply:
@nirhutnik - having it just classification/regression is weird for a label type

Reply:
at least let's replace the space with _